### ISSUE TITLE
PollingMode AutoPoll in combination with pollIntervalSeconds Number.MAX_SAFE_INTEGER leads to endless polling loop

### ISSUE THREAD

**Issue by dataGeeek** [configcat/react-sdk#29]
### Describe the bug

I tried to setup configCat in my application with minimal auto-loading and used the following config:

```           
 <ConfigCatProvider
                sdkKey={config.configCat.apiKey}
                pollingMode={PollingMode.AutoPoll}
                options={{
                    pollIntervalSeconds: Number.MAX_SAFE_INTEGER,
                    dataGovernance: DataGovernance.EuOnly,
                    logger: createConsoleLogger(LogLevel.Off),
                }}
            >
```

When using this config, we see multiple config requests per second

### To reproduce

Setup configCat with version 2.2.1 in a react application with the above-supplied config.

### Expected behavior

I would expect, that the config is polled from your backend as little as possible

### Screenshots

**Comment by laliconfigcat** [comment]
Hello @dataGeeek ,

Thanks for reporting this issue. We are checking it and we will get back to you.

**Comment by laliconfigcat** [comment]
Hello @dataGeeek,

You are totally right, the underlying setTimeout cannot handle Number.MAX_SAFE_INTEGER as the interval, its maximum value is 2147483647. 
I added a ticket to our backlog to put some extra validation to the pollIntervalSeconds and update this wrong information in all of our documentations that states that the max pollIntervalSeconds is Number.MAX_SAFE_INTEGER.
Thanks for your report and we are sorry for the inconvenience. 
Until it is fixed we'd recommend you to use a value < 2147483647 / 1000 as pollIntervalSeconds.

**Comment by adams85** [comment]
Closing as fixed & released in v3.0.0.
